From: Gregory CLEMENT <gregory.clement@free-electrons.com>
Date: Fri, 10 Jun 2016 15:23:29 +0200
Subject: arm64: marvell: enable Armada 3700 clock drivers
Git-commit: ff60d834d1e0d1b4560b14daab1bc165667af7b4
Patch-mainline: v4.8-rc1
References: fate#319899 fate#320700

This patch enables the support for the clocks drivers used on the
Armada 3700.

Signed-off-by: Gregory CLEMENT <gregory.clement@free-electrons.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/Kconfig.platforms |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -60,6 +60,7 @@ config ARCH_MEDIATEK
 
 config ARCH_MVEBU
 	bool "Marvell EBU SoC Family"
+	select ARMADA_37XX_CLK
 	help
 	  This enables support for Marvell EBU family such as the
 	  Armada 3700 SoC family.
